Time taken to run the queries on ./repositories/Django/thread13/Amsterdam_signals/Amsterdam-signals-894c7e9: 2454.727 seconds.
Time taken to run the queries on ./repositories/Django/thread13/ArchiveBox_ArchiveBox/ArchiveBox-ArchiveBox-1d49bee: 2393.428 seconds.
Time taken to run the queries on ./repositories/Django/thread13/Archmonger_Conreq/Archmonger-Conreq-8007f5e: 2033.196 seconds.
Time taken to run the queries on ./repositories/Django/thread13/BenoitPrmt_Monod-Crush/BenoitPrmt-Monod-Crush-c290005: 2264.94 seconds.
Time taken to run the queries on ./repositories/Django/thread13/CMSgov_bluebutton-web-server/CMSgov-bluebutton-web-server-df66443: 2538.131 seconds.
Time taken to run the queries on ./repositories/Django/thread13/CanaimaKueka_tribus/CanaimaKueka-tribus-244bc10: 2267.513 seconds.
Analysis failed for: ./repositories/Django/thread13/CxSci_atlas-of-opportunity/CxSci-atlas-of-opportunity-6eff07b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/CxSci_atlas-of-opportunity/CxSci-atlas-of-opportunity-6eff07b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 3m2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/CxSci_atlas-of-opportunity/CxSci-atlas-of-opportunity-6eff07b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/CxSci_atlas-of-opportunity/CxSci-atlas-of-opportunity-6eff07b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/CxSci_atlas-of-opportunity/CxSci-atlas-of-opportunity-6eff07b: 367.205 seconds.
Analysis failed for: ./repositories/Django/thread13/DemocracyClub_yournextrepresentative/DemocracyClub-yournextrepresentative-cef245c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/DemocracyClub_yournextrepresentative/DemocracyClub-yournextrepresentative-cef245c -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m43s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/DemocracyClub_yournextrepresentative/DemocracyClub-yournextrepresentative-cef245c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/DemocracyClub_yournextrepresentative/DemocracyClub-yournextrepresentative-cef245c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/DemocracyClub_yournextrepresentative/DemocracyClub-yournextrepresentative-cef245c: 262.465 seconds.
Time taken to run the queries on ./repositories/Django/thread13/ErasmusMC-Bioinformatics_WeFaceNano/ErasmusMC-Bioinformatics-WeFaceNano-84ad29c: 1950.504 seconds.
Analysis failed for: ./repositories/Django/thread13/GraemeHawker_GBEnergyDataManager/GraemeHawker-GBEnergyDataManager-9de5de0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/GraemeHawker_GBEnergyDataManager/GraemeHawker-GBEnergyDataManager-9de5de0 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/GraemeHawker_GBEnergyDataManager/GraemeHawker-GBEnergyDataManager-9de5de0-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/GraemeHawker_GBEnergyDataManager/GraemeHawker-GBEnergyDataManager-9de5de0-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/GraemeHawker_GBEnergyDataManager/GraemeHawker-GBEnergyDataManager-9de5de0: 298.093 seconds.
Time taken to run the queries on ./repositories/Django/thread13/HyphaApp_hypha/HyphaApp-hypha-3304ec4: 2765.609 seconds.
Time taken to run the queries on ./repositories/Django/thread13/Inboxen_Inboxen/Inboxen-Inboxen-30c9d9a: 2500.829 seconds.
Analysis failed for: ./repositories/Django/thread13/InspectorIncognito_gtfs-editor-backend/InspectorIncognito-gtfs-editor-backend-3c368d0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/InspectorIncognito_gtfs-editor-backend/InspectorIncognito-gtfs-editor-backend-3c368d0 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/InspectorIncognito_gtfs-editor-backend/InspectorIncognito-gtfs-editor-backend-3c368d0-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/InspectorIncognito_gtfs-editor-backend/InspectorIncognito-gtfs-editor-backend-3c368d0-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/InspectorIncognito_gtfs-editor-backend/InspectorIncognito-gtfs-editor-backend-3c368d0: 199.231 seconds.
Analysis failed for: ./repositories/Django/thread13/JasonSanford_geojsonlint.com/JasonSanford-geojsonlint.com-a986fce Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/JasonSanford_geojsonlint.com/JasonSanford-geojsonlint.com-a986fce -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/JasonSanford_geojsonlint.com/JasonSanford-geojsonlint.com-a986fce-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/JasonSanford_geojsonlint.com/JasonSanford-geojsonlint.com-a986fce-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/JasonSanford_geojsonlint.com/JasonSanford-geojsonlint.com-a986fce: 192.881 seconds.
Time taken to run the queries on ./repositories/Django/thread13/Jumpi96_helpo/Jumpi96-helpo-b4f342d: 2627.723 seconds.
Time taken to run the queries on ./repositories/Django/thread13/Kelketek_artconomy/Kelketek-artconomy-2cc2167: 2708.747 seconds.
Analysis failed for: ./repositories/Django/thread13/Me-Adota_website/Me-Adota-website-4081899 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/Me-Adota_website/Me-Adota-website-4081899 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/Me-Adota_website/Me-Adota-website-4081899-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/Me-Adota_website/Me-Adota-website-4081899-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/Me-Adota_website/Me-Adota-website-4081899: 180.707 seconds.
Analysis failed for: ./repositories/Django/thread13/MedTAG_medtag-core/MedTAG-medtag-core-92b9914 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/MedTAG_medtag-core/MedTAG-medtag-core-92b9914 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m33s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/MedTAG_medtag-core/MedTAG-medtag-core-92b9914-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/MedTAG_medtag-core/MedTAG-medtag-core-92b9914-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/MedTAG_medtag-core/MedTAG-medtag-core-92b9914: 315.562 seconds.
Time taken to run the queries on ./repositories/Django/thread13/NFM-Studios_project-olly/NFM-Studios-project-olly-3e7d394: 2316.127 seconds.
Time taken to run the queries on ./repositories/Django/thread13/PGPerfFarm_pgperffarm_server/PGPerfFarm-pgperffarm_server-8f11c7b: 2312.452 seconds.
Time taken to run the queries on ./repositories/Django/thread13/ParthKalkar_RusLearn/ParthKalkar-RusLearn-cecc00e: 1988.344 seconds.
Time taken to run the queries on ./repositories/Django/thread13/PrivacyScore_PrivacyScore/PrivacyScore-PrivacyScore-fd64bf5: 2239.904 seconds.
Time taken to run the queries on ./repositories/Django/thread13/ProjectDossier_cruise-screening/ProjectDossier-cruise-screening-10f6a28: 2596.168 seconds.
Time taken to run the queries on ./repositories/Django/thread13/PyAr_asoc_members/PyAr-asoc_members-7189fce: 2128.037 seconds.
Analysis failed for: ./repositories/Django/thread13/ShahriyarR_one-time-secret-share/ShahriyarR-one-time-secret-share-3ae0330 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/ShahriyarR_one-time-secret-share/ShahriyarR-one-time-secret-share-3ae0330 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m20s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/ShahriyarR_one-time-secret-share/ShahriyarR-one-time-secret-share-3ae0330-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/ShahriyarR_one-time-secret-share/ShahriyarR-one-time-secret-share-3ae0330-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/ShahriyarR_one-time-secret-share/ShahriyarR-one-time-secret-share-3ae0330: 214.414 seconds.
Time taken to run the queries on ./repositories/Django/thread13/TencentBlueKing_bk-user/TencentBlueKing-bk-user-06f4bf8: 2505.82 seconds.
Analysis failed for: ./repositories/Django/thread13/TencentBlueKing_blueking-dbm/TencentBlueKing-blueking-dbm-dc8e472 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/TencentBlueKing_blueking-dbm/TencentBlueKing-blueking-dbm-dc8e472 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m36s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/TencentBlueKing_blueking-dbm/TencentBlueKing-blueking-dbm-dc8e472-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 29.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/TencentBlueKing_blueking-dbm/TencentBlueKing-blueking-dbm-dc8e472-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/TencentBlueKing_blueking-dbm/TencentBlueKing-blueking-dbm-dc8e472: 371.807 seconds.
Time taken to run the queries on ./repositories/Django/thread13/TimSC_pycrocosm/TimSC-pycrocosm-c1509a3: 2282.46 seconds.
Analysis failed for: ./repositories/Django/thread13/Trinkle23897_tuixue.online-visa/Trinkle23897-tuixue.online-visa-31bc185 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/Trinkle23897_tuixue.online-visa/Trinkle23897-tuixue.online-visa-31bc185 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/Trinkle23897_tuixue.online-visa/Trinkle23897-tuixue.online-visa-31bc185-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/Trinkle23897_tuixue.online-visa/Trinkle23897-tuixue.online-visa-31bc185-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/Trinkle23897_tuixue.online-visa/Trinkle23897-tuixue.online-visa-31bc185: 312.211 seconds.
Analysis failed for: ./repositories/Django/thread13/TrivadisPF_platys-modern-data-platform/TrivadisPF-platys-modern-data-platform-c0c221b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/TrivadisPF_platys-modern-data-platform/TrivadisPF-platys-modern-data-platform-c0c221b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 56.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/TrivadisPF_platys-modern-data-platform/TrivadisPF-platys-modern-data-platform-c0c221b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/TrivadisPF_platys-modern-data-platform/TrivadisPF-platys-modern-data-platform-c0c221b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/TrivadisPF_platys-modern-data-platform/TrivadisPF-platys-modern-data-platform-c0c221b: 180.702 seconds.
Analysis failed for: ./repositories/Django/thread13/UmSenhorQualquer_pyforms-web/UmSenhorQualquer-pyforms-web-4cf1d71 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/UmSenhorQualquer_pyforms-web/UmSenhorQualquer-pyforms-web-4cf1d71 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/UmSenhorQualquer_pyforms-web/UmSenhorQualquer-pyforms-web-4cf1d71-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/UmSenhorQualquer_pyforms-web/UmSenhorQualquer-pyforms-web-4cf1d71-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/UmSenhorQualquer_pyforms-web/UmSenhorQualquer-pyforms-web-4cf1d71: 296.962 seconds.
Time taken to run the queries on ./repositories/Django/thread13/World-Konkani-Centre_sushiksha-website/World-Konkani-Centre-sushiksha-website-053e7c7: 2655.815 seconds.
Time taken to run the queries on ./repositories/Django/thread13/YAVGroup_Verilog-OJ/YAVGroup-Verilog-OJ-d01eaa4: 2368.582 seconds.
Analysis failed for: ./repositories/Django/thread13/amindeed_Gmail-AutoResponder/amindeed-Gmail-AutoResponder-a8a3eb1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/amindeed_Gmail-AutoResponder/amindeed-Gmail-AutoResponder-a8a3eb1 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/amindeed_Gmail-AutoResponder/amindeed-Gmail-AutoResponder-a8a3eb1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/amindeed_Gmail-AutoResponder/amindeed-Gmail-AutoResponder-a8a3eb1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/amindeed_Gmail-AutoResponder/amindeed-Gmail-AutoResponder-a8a3eb1: 185.341 seconds.
Analysis failed for: ./repositories/Django/thread13/amyxzhang_wikum/amyxzhang-wikum-23ee7cd Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/amyxzhang_wikum/amyxzhang-wikum-23ee7cd -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 3m15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/amyxzhang_wikum/amyxzhang-wikum-23ee7cd-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 24.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/amyxzhang_wikum/amyxzhang-wikum-23ee7cd-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/amyxzhang_wikum/amyxzhang-wikum-23ee7cd: 402.024 seconds.
Time taken to run the queries on ./repositories/Django/thread13/andwang1_BrainSurfaceTK/andwang1-BrainSurfaceTK-d043d5b: 2656.309 seconds.
Time taken to run the queries on ./repositories/Django/thread13/apmilen_homenet/apmilen-homenet-688ff89: 2123.152 seconds.
Analysis failed for: ./repositories/Django/thread13/arxiv-vanity_arxiv-vanity/arxiv-vanity-arxiv-vanity-f7eb2f1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/arxiv-vanity_arxiv-vanity/arxiv-vanity-arxiv-vanity-f7eb2f1 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/arxiv-vanity_arxiv-vanity/arxiv-vanity-arxiv-vanity-f7eb2f1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/arxiv-vanity_arxiv-vanity/arxiv-vanity-arxiv-vanity-f7eb2f1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/arxiv-vanity_arxiv-vanity/arxiv-vanity-arxiv-vanity-f7eb2f1: 222.715 seconds.
Analysis failed for: ./repositories/Django/thread13/barberscore_barberscore-api/barberscore-barberscore-api-bef571c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/barberscore_barberscore-api/barberscore-barberscore-api-bef571c -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m23s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/barberscore_barberscore-api/barberscore-barberscore-api-bef571c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/barberscore_barberscore-api/barberscore-barberscore-api-bef571c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/barberscore_barberscore-api/barberscore-barberscore-api-bef571c: 232.639 seconds.
Analysis failed for: ./repositories/Django/thread13/bkfox_aircox/bkfox-aircox-30ffff9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/bkfox_aircox/bkfox-aircox-30ffff9 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/bkfox_aircox/bkfox-aircox-30ffff9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/bkfox_aircox/bkfox-aircox-30ffff9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/bkfox_aircox/bkfox-aircox-30ffff9: 173.917 seconds.
Time taken to run the queries on ./repositories/Django/thread13/cepdnaclk_e16-3yp-smart-vending-machine/cepdnaclk-e16-3yp-smart-vending-machine-63ad4e9: 2364.139 seconds.
Time taken to run the queries on ./repositories/Django/thread13/certsocietegenerale_FIR/certsocietegenerale-FIR-463675e: 2002.491 seconds.
Analysis failed for: ./repositories/Django/thread13/chaincodelabs_bitcoinperf/chaincodelabs-bitcoinperf-0a1d6f5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/chaincodelabs_bitcoinperf/chaincodelabs-bitcoinperf-0a1d6f5 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/chaincodelabs_bitcoinperf/chaincodelabs-bitcoinperf-0a1d6f5-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/chaincodelabs_bitcoinperf/chaincodelabs-bitcoinperf-0a1d6f5-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/chaincodelabs_bitcoinperf/chaincodelabs-bitcoinperf-0a1d6f5: 314.29 seconds.
Analysis failed for: ./repositories/Django/thread13/chandusanjith_E-Commerce-Condiments-shop/chandusanjith-E-Commerce-Condiments-shop-c67971c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/chandusanjith_E-Commerce-Condiments-shop/chandusanjith-E-Commerce-Condiments-shop-c67971c -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/chandusanjith_E-Commerce-Condiments-shop/chandusanjith-E-Commerce-Condiments-shop-c67971c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/chandusanjith_E-Commerce-Condiments-shop/chandusanjith-E-Commerce-Condiments-shop-c67971c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/chandusanjith_E-Commerce-Condiments-shop/chandusanjith-E-Commerce-Condiments-shop-c67971c: 195.989 seconds.
Analysis failed for: ./repositories/Django/thread13/chembl_GLaDOS/chembl-GLaDOS-044ed5f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/chembl_GLaDOS/chembl-GLaDOS-044ed5f -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m35s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/chembl_GLaDOS/chembl-GLaDOS-044ed5f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/chembl_GLaDOS/chembl-GLaDOS-044ed5f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/chembl_GLaDOS/chembl-GLaDOS-044ed5f: 240.844 seconds.
Time taken to run the queries on ./repositories/Django/thread13/cjlee112_socraticqs2/cjlee112-socraticqs2-2e7dd9d: 2456.945 seconds.
Analysis failed for: ./repositories/Django/thread13/climu_openstudyroom/climu-openstudyroom-f05941f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/climu_openstudyroom/climu-openstudyroom-f05941f -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m20s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/climu_openstudyroom/climu-openstudyroom-f05941f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/climu_openstudyroom/climu-openstudyroom-f05941f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/climu_openstudyroom/climu-openstudyroom-f05941f: 227.782 seconds.
Analysis failed for: ./repositories/Django/thread13/collabovid_collabovid/collabovid-collabovid-e093e9b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/collabovid_collabovid/collabovid-collabovid-e093e9b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m41s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/collabovid_collabovid/collabovid-collabovid-e093e9b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/collabovid_collabovid/collabovid-collabovid-e093e9b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/collabovid_collabovid/collabovid-collabovid-e093e9b: 354.792 seconds.
Analysis failed for: ./repositories/Django/thread13/coronasafe_ayushma/coronasafe-ayushma-f8bb89c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/coronasafe_ayushma/coronasafe-ayushma-f8bb89c -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/coronasafe_ayushma/coronasafe-ayushma-f8bb89c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/coronasafe_ayushma/coronasafe-ayushma-f8bb89c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/coronasafe_ayushma/coronasafe-ayushma-f8bb89c: 308.345 seconds.
Time taken to run the queries on ./repositories/Django/thread13/d9pouces_df_config/d9pouces-df_config-5c06160: 2026.973 seconds.
Analysis failed for: ./repositories/Django/thread13/databat-io_sonar/databat-io-sonar-1df6256 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/databat-io_sonar/databat-io-sonar-1df6256 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/databat-io_sonar/databat-io-sonar-1df6256-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/databat-io_sonar/databat-io-sonar-1df6256-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/databat-io_sonar/databat-io-sonar-1df6256: 223.532 seconds.
Analysis failed for: ./repositories/Django/thread13/datalets_public-health-ch/datalets-public-health-ch-1cf81f0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/datalets_public-health-ch/datalets-public-health-ch-1cf81f0 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m25s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/datalets_public-health-ch/datalets-public-health-ch-1cf81f0-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/datalets_public-health-ch/datalets-public-health-ch-1cf81f0-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/datalets_public-health-ch/datalets-public-health-ch-1cf81f0: 220.963 seconds.
Time taken to run the queries on ./repositories/Django/thread13/digitalfabrik_opendrift-leeway-webgui/digitalfabrik-opendrift-leeway-webgui-bb3b2b9: 2652.718 seconds.
Analysis failed for: ./repositories/Django/thread13/digitalfox_pydici/digitalfox-pydici-a1d78c6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/digitalfox_pydici/digitalfox-pydici-a1d78c6 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/digitalfox_pydici/digitalfox-pydici-a1d78c6-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 21.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/digitalfox_pydici/digitalfox-pydici-a1d78c6-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/digitalfox_pydici/digitalfox-pydici-a1d78c6: 309.409 seconds.
Analysis failed for: ./repositories/Django/thread13/django-daiquiri_daiquiri/django-daiquiri-daiquiri-5ae2a14 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/django-daiquiri_daiquiri/django-daiquiri-daiquiri-5ae2a14 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m39s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/django-daiquiri_daiquiri/django-daiquiri-daiquiri-5ae2a14-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/django-daiquiri_daiquiri/django-daiquiri-daiquiri-5ae2a14-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/django-daiquiri_daiquiri/django-daiquiri-daiquiri-5ae2a14: 257.825 seconds.
Analysis failed for: ./repositories/Django/thread13/earthly_website/earthly-website-1dfe966 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/earthly_website/earthly-website-1dfe966 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/earthly_website/earthly-website-1dfe966-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/earthly_website/earthly-website-1dfe966-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/earthly_website/earthly-website-1dfe966: 290.511 seconds.
Analysis failed for: ./repositories/Django/thread13/ecobasa_ecobasa/ecobasa-ecobasa-e6c0838 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/ecobasa_ecobasa/ecobasa-ecobasa-e6c0838 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/ecobasa_ecobasa/ecobasa-ecobasa-e6c0838-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/ecobasa_ecobasa/ecobasa-ecobasa-e6c0838-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/ecobasa_ecobasa/ecobasa-ecobasa-e6c0838: 202.823 seconds.
Analysis failed for: ./repositories/Django/thread13/eduzen_website/eduzen-website-5a002f3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/eduzen_website/eduzen-website-5a002f3 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m21s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/eduzen_website/eduzen-website-5a002f3-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/eduzen_website/eduzen-website-5a002f3-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/eduzen_website/eduzen-website-5a002f3: 227.463 seconds.
Time taken to run the queries on ./repositories/Django/thread13/evcloud_evcloud/evcloud-evcloud-07315ff: 2066.37 seconds.
Analysis failed for: ./repositories/Django/thread13/grapesmoker_qems2/grapesmoker-qems2-84930d6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/grapesmoker_qems2/grapesmoker-qems2-84930d6 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 58.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/grapesmoker_qems2/grapesmoker-qems2-84930d6-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/grapesmoker_qems2/grapesmoker-qems2-84930d6-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/grapesmoker_qems2/grapesmoker-qems2-84930d6: 182.98 seconds.
Analysis failed for: ./repositories/Django/thread13/gtech-mulearn_mulearnbackend/gtech-mulearn-mulearnbackend-6f2eff3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/gtech-mulearn_mulearnbackend/gtech-mulearn-mulearnbackend-6f2eff3 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m35s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/gtech-mulearn_mulearnbackend/gtech-mulearn-mulearnbackend-6f2eff3-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/gtech-mulearn_mulearnbackend/gtech-mulearn-mulearnbackend-6f2eff3-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/gtech-mulearn_mulearnbackend/gtech-mulearn-mulearnbackend-6f2eff3: 226.763 seconds.
Time taken to run the queries on ./repositories/Django/thread13/guohongze_adminset/guohongze-adminset-340de2d: 2398.438 seconds.
Time taken to run the queries on ./repositories/Django/thread13/hydroshare_hydroshare/hydroshare-hydroshare-742fbce: 3416.022 seconds.
Time taken to run the queries on ./repositories/Django/thread13/inventree_InvenTree/inventree-InvenTree-c2a9b02: 3045.158 seconds.
Time taken to run the queries on ./repositories/Django/thread13/jamesvandyne_tanzawa/jamesvandyne-tanzawa-8032175: 2477.453 seconds.
Analysis failed for: ./repositories/Django/thread13/jberghoef_wagtail-tag-manager/jberghoef-wagtail-tag-manager-7ef67f0 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/jberghoef_wagtail-tag-manager/jberghoef-wagtail-tag-manager-7ef67f0 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m22s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/jberghoef_wagtail-tag-manager/jberghoef-wagtail-tag-manager-7ef67f0-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/jberghoef_wagtail-tag-manager/jberghoef-wagtail-tag-manager-7ef67f0-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/jberghoef_wagtail-tag-manager/jberghoef-wagtail-tag-manager-7ef67f0: 229.576 seconds.
Time taken to run the queries on ./repositories/Django/thread13/jittat_admapp/jittat-admapp-96b4e2d: 2574.549 seconds.
Analysis failed for: ./repositories/Django/thread13/juliarobles_graffitisWeb/juliarobles-graffitisWeb-ddf5e2b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/juliarobles_graffitisWeb/juliarobles-graffitisWeb-ddf5e2b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m20s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/juliarobles_graffitisWeb/juliarobles-graffitisWeb-ddf5e2b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/juliarobles_graffitisWeb/juliarobles-graffitisWeb-ddf5e2b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/juliarobles_graffitisWeb/juliarobles-graffitisWeb-ddf5e2b: 215.679 seconds.
Analysis failed for: ./repositories/Django/thread13/junlee91_instagram-clone/junlee91-instagram-clone-b66c468 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/junlee91_instagram-clone/junlee91-instagram-clone-b66c468 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m19s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/junlee91_instagram-clone/junlee91-instagram-clone-b66c468-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/junlee91_instagram-clone/junlee91-instagram-clone-b66c468-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/junlee91_instagram-clone/junlee91-instagram-clone-b66c468: 207.214 seconds.
Analysis failed for: ./repositories/Django/thread13/kavania2002_MobiMart/kavania2002-MobiMart-1c2d1f5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/kavania2002_MobiMart/kavania2002-MobiMart-1c2d1f5 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 55.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kavania2002_MobiMart/kavania2002-MobiMart-1c2d1f5-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kavania2002_MobiMart/kavania2002-MobiMart-1c2d1f5-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/kavania2002_MobiMart/kavania2002-MobiMart-1c2d1f5: 182.898 seconds.
Analysis failed for: ./repositories/Django/thread13/kiwitcms_tenants/kiwitcms-tenants-98c5046 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/kiwitcms_tenants/kiwitcms-tenants-98c5046 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m11s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kiwitcms_tenants/kiwitcms-tenants-98c5046-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kiwitcms_tenants/kiwitcms-tenants-98c5046-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/kiwitcms_tenants/kiwitcms-tenants-98c5046: 195.979 seconds.
Analysis failed for: ./repositories/Django/thread13/kkosiba_ecommerce-backend/kkosiba-ecommerce-backend-a0c05f9 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/kkosiba_ecommerce-backend/kkosiba-ecommerce-backend-a0c05f9 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kkosiba_ecommerce-backend/kkosiba-ecommerce-backend-a0c05f9-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kkosiba_ecommerce-backend/kkosiba-ecommerce-backend-a0c05f9-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/kkosiba_ecommerce-backend/kkosiba-ecommerce-backend-a0c05f9: 203.192 seconds.
Analysis failed for: ./repositories/Django/thread13/kyoung_unconf/kyoung-unconf-8117d0b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/kyoung_unconf/kyoung-unconf-8117d0b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kyoung_unconf/kyoung-unconf-8117d0b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 6.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/kyoung_unconf/kyoung-unconf-8117d0b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/kyoung_unconf/kyoung-unconf-8117d0b: 175.873 seconds.
Analysis failed for: ./repositories/Django/thread13/laike9m_My_Blog/laike9m-My_Blog-e9d391b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/laike9m_My_Blog/laike9m-My_Blog-e9d391b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/laike9m_My_Blog/laike9m-My_Blog-e9d391b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/laike9m_My_Blog/laike9m-My_Blog-e9d391b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/laike9m_My_Blog/laike9m-My_Blog-e9d391b: 191.994 seconds.
Analysis failed for: ./repositories/Django/thread13/lingster_drf-api-tracking/lingster-drf-api-tracking-e52c365 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/lingster_drf-api-tracking/lingster-drf-api-tracking-e52c365 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/lingster_drf-api-tracking/lingster-drf-api-tracking-e52c365-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/lingster_drf-api-tracking/lingster-drf-api-tracking-e52c365-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/lingster_drf-api-tracking/lingster-drf-api-tracking-e52c365: 191.212 seconds.
Time taken to run the queries on ./repositories/Django/thread13/melizeche_ayudapy/melizeche-ayudapy-e2657d0: 2038.613 seconds.
Time taken to run the queries on ./repositories/Django/thread13/michaelhenry_Localizr/michaelhenry-Localizr-23a38d5: 2006.926 seconds.
Time taken to run the queries on ./repositories/Django/thread13/mjtamlyn_tamlynscore/mjtamlyn-tamlynscore-6ac38d9: 2066.726 seconds.
Analysis failed for: ./repositories/Django/thread13/moyada_stealer/moyada-stealer-5f3330d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/moyada_stealer/moyada-stealer-5f3330d -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m20s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/moyada_stealer/moyada-stealer-5f3330d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/moyada_stealer/moyada-stealer-5f3330d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/moyada_stealer/moyada-stealer-5f3330d: 212.648 seconds.
Analysis failed for: ./repositories/Django/thread13/mradamcox_ohmg/mradamcox-ohmg-64cf1d7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/mradamcox_ohmg/mradamcox-ohmg-64cf1d7 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/mradamcox_ohmg/mradamcox-ohmg-64cf1d7-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 20.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/mradamcox_ohmg/mradamcox-ohmg-64cf1d7-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/mradamcox_ohmg/mradamcox-ohmg-64cf1d7: 319.668 seconds.
Time taken to run the queries on ./repositories/Django/thread13/neuroforgede_nfcompose/neuroforgede-nfcompose-512bfe2: 2784.925 seconds.
Time taken to run the queries on ./repositories/Django/thread13/neutrons_data_workflow/neutrons-data_workflow-3a2a5b6: 2321.162 seconds.
Time taken to run the queries on ./repositories/Django/thread13/nsupdate-info_nsupdate.info/nsupdate-info-nsupdate.info-c15f04b: 2413.101 seconds.
Time taken to run the queries on ./repositories/Django/thread13/oser-cs_oser-backend/oser-cs-oser-backend-ac5f18d: 2096.541 seconds.
Time taken to run the queries on ./repositories/Django/thread13/peeringdb_peeringdb/peeringdb-peeringdb-85ae7bc: 2590.287 seconds.
Time taken to run the queries on ./repositories/Django/thread13/perfguru87_panopticum/perfguru87-panopticum-9249459: 2068.103 seconds.
Analysis failed for: ./repositories/Django/thread13/pinax_pinax-badges/pinax-pinax-badges-9f3895b Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/pinax_pinax-badges/pinax-pinax-badges-9f3895b -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m12s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/pinax_pinax-badges/pinax-pinax-badges-9f3895b-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/pinax_pinax-badges/pinax-pinax-badges-9f3895b-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/pinax_pinax-badges/pinax-pinax-badges-9f3895b: 192.409 seconds.
Analysis failed for: ./repositories/Django/thread13/pinax_pinax-blog/pinax-pinax-blog-b6d8755 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/pinax_pinax-blog/pinax-pinax-blog-b6d8755 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 59.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/pinax_pinax-blog/pinax-pinax-blog-b6d8755-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/pinax_pinax-blog/pinax-pinax-blog-b6d8755-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/pinax_pinax-blog/pinax-pinax-blog-b6d8755: 187.16 seconds.
Time taken to run the queries on ./repositories/Django/thread13/postgres_pgweb/postgres-pgweb-88cdd6b: 2325.242 seconds.
Time taken to run the queries on ./repositories/Django/thread13/raveberry_raveberry/raveberry-raveberry-7b73805: 2378.633 seconds.
Time taken to run the queries on ./repositories/Django/thread13/rohini-omnilytiqs_vmwellness/rohini-omnilytiqs-vmwellness-915392b: 2003.375 seconds.
Analysis failed for: ./repositories/Django/thread13/roks0n_podcasti.si/roks0n-podcasti.si-f578005 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/roks0n_podcasti.si/roks0n-podcasti.si-f578005 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m24s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/roks0n_podcasti.si/roks0n-podcasti.si-f578005-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/roks0n_podcasti.si/roks0n-podcasti.si-f578005-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/roks0n_podcasti.si/roks0n-podcasti.si-f578005: 221.33 seconds.
Analysis failed for: ./repositories/Django/thread13/rukzuk_rukzuk/rukzuk-rukzuk-41d84a6 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/rukzuk_rukzuk/rukzuk-rukzuk-41d84a6 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/rukzuk_rukzuk/rukzuk-rukzuk-41d84a6-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/rukzuk_rukzuk/rukzuk-rukzuk-41d84a6-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/rukzuk_rukzuk/rukzuk-rukzuk-41d84a6: 179.803 seconds.
Analysis failed for: ./repositories/Django/thread13/shabalin13_Arithmetic-PvP/shabalin13-Arithmetic-PvP-da08377 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/shabalin13_Arithmetic-PvP/shabalin13-Arithmetic-PvP-da08377 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/shabalin13_Arithmetic-PvP/shabalin13-Arithmetic-PvP-da08377-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/shabalin13_Arithmetic-PvP/shabalin13-Arithmetic-PvP-da08377-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/shabalin13_Arithmetic-PvP/shabalin13-Arithmetic-PvP-da08377: 194.079 seconds.
Time taken to run the queries on ./repositories/Django/thread13/sissbruecker_linkding/sissbruecker-linkding-77e1525: 2401.058 seconds.
Analysis failed for: ./repositories/Django/thread13/studyhub-co_physics-is-beautiful/studyhub-co-physics-is-beautiful-57e1ab1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/studyhub-co_physics-is-beautiful/studyhub-co-physics-is-beautiful-57e1ab1 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/studyhub-co_physics-is-beautiful/studyhub-co-physics-is-beautiful-57e1ab1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 21.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/studyhub-co_physics-is-beautiful/studyhub-co-physics-is-beautiful-57e1ab1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/studyhub-co_physics-is-beautiful/studyhub-co-physics-is-beautiful-57e1ab1: 305.06 seconds.
Analysis failed for: ./repositories/Django/thread13/thomas-yanxin_Smart_container/thomas-yanxin-ColugoMum-6b9704e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/thomas-yanxin_Smart_container/thomas-yanxin-ColugoMum-6b9704e -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 3m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/thomas-yanxin_Smart_container/thomas-yanxin-ColugoMum-6b9704e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 19.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/thomas-yanxin_Smart_container/thomas-yanxin-ColugoMum-6b9704e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/thomas-yanxin_Smart_container/thomas-yanxin-ColugoMum-6b9704e: 417.11 seconds.
Time taken to run the queries on ./repositories/Django/thread13/tnpfldyd_semi_pjt_2/tnpfldyd-semi_pjt_2-67c26c9: 2202.019 seconds.
Analysis failed for: ./repositories/Django/thread13/tough-dev-school_education-backend/tough-dev-school-education-backend-ab526e4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/tough-dev-school_education-backend/tough-dev-school-education-backend-ab526e4 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m32s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/tough-dev-school_education-backend/tough-dev-school-education-backend-ab526e4-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/tough-dev-school_education-backend/tough-dev-school-education-backend-ab526e4-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/tough-dev-school_education-backend/tough-dev-school-education-backend-ab526e4: 244.868 seconds.
Time taken to run the queries on ./repositories/Django/thread13/transport-nantes_tn_web/transport-nantes-tn_web-8cac61f: 2252.805 seconds.
Analysis failed for: ./repositories/Django/thread13/tveastman_secateur/tveastman-secateur-225e20d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/tveastman_secateur/tveastman-secateur-225e20d -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/tveastman_secateur/tveastman-secateur-225e20d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/tveastman_secateur/tveastman-secateur-225e20d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/tveastman_secateur/tveastman-secateur-225e20d: 234.305 seconds.
Analysis failed for: ./repositories/Django/thread13/uktrade_directory-cms/uktrade-directory-cms-7382fc1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/uktrade_directory-cms/uktrade-directory-cms-7382fc1 -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/uktrade_directory-cms/uktrade-directory-cms-7382fc1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/uktrade_directory-cms/uktrade-directory-cms-7382fc1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/uktrade_directory-cms/uktrade-directory-cms-7382fc1: 238.213 seconds.
Time taken to run the queries on ./repositories/Django/thread13/uno-isqa-8950_fitgirl-inc/uno-isqa-8950-fitgirl-inc-2656e73: 1861.175 seconds.
Analysis failed for: ./repositories/Django/thread13/wahaj_Hamzar/wahaj-Hamzar-3dfec7c Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread13/wahaj_Hamzar/wahaj-Hamzar-3dfec7c -l=python -t=11 -ct=13
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 57.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/wahaj_Hamzar/wahaj-Hamzar-3dfec7c-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 7.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/wahaj_Hamzar/wahaj-Hamzar-3dfec7c-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread13/wahaj_Hamzar/wahaj-Hamzar-3dfec7c: 158.617 seconds.
Analysis failed for: ./repositories/Django/thread13/ytsaurus_ytsaurus/ytsaurus-ytsaurus-4fc8232 Reason: Error: spawnSync /bin/sh ENOBUFS
Could not delete database for: ./repositories/Django/thread13/ytsaurus_ytsaurus/ytsaurus-ytsaurus-4fc8232 Reason: Error: ENOTEMPTY: directory not empty, rmdir './repositories/Django/thread13/ytsaurus_ytsaurus/ytsaurus-ytsaurus-4fc8232-database/src/home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread13/ytsaurus_ytsaurus/ytsaurus-ytsaurus-4fc8232/contrib/python'
Time taken to run the queries on ./repositories/Django/thread13/ytsaurus_ytsaurus/ytsaurus-ytsaurus-4fc8232: 182.633 seconds.
Time taken to run the queries on ./repositories/Django/thread13/z-station_cappa/z-station-cappa-14ed9fb: 1815.722 seconds.
Time taken to run the queries on ./repositories/Django/thread13/zmister2016_MrDoc/zmister2016-MrDoc-436d728: 2113.984 seconds.
Time taken to run the queries on ./repositories/Django/thread13/zuri-training_excel_comp_team_106/zuri-training-excel_comp_team_106-9d3fe30: 1630.712 seconds.
Time taken to run the queries and generate the statistics: 136805.857 seconds.
